examples/LLM_Workflows/NER_Example/run.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+
+import lancedb_module
+import ner_extraction
+
+from hamilton import driver, lifecycle
+
+
+def build_driver(adapter_list):
+    """Builds the driver with the necessary modules and adapters."""
+    dr = (
+        driver.Builder()
+        .with_config({})
+        .with_modules(ner_extraction, lancedb_module)
+        .with_adapters(*adapter_list)
+        .build()
+    )
+    return dr
+
+
+def load_data(table_name: str, use_tracker: bool = False):
+    adapter_list = [lifecycle.PrintLn()]
+    if use_tracker:
+        from hamilton_sdk import adapters
+
+        tracker = adapters.HamiltonTracker(
+            project_id=41,  # modify this as needed
+            username="elijah@dagworks.io",
+            dag_name="ner-lancedb-pipeline",
+            tags={"context": "extraction", "team": "MY_TEAM", "version": "1"},
+        )
+        adapter_list.append(tracker)
+
+    dr = build_driver(adapter_list)
+    # display the graph
+    dr.display_all_functions("ner_extraction_pipeline.png")
+
+    results = dr.execute(
+        ["load_into_lancedb"],
+        inputs={"table_name": table_name},
+    )
+    print(results)
+
+
+def query_data(query: str, table_name: str, use_tracker: bool = False):
+    adapter_list = [lifecycle.PrintLn()]
+    if use_tracker:
+        from hamilton_sdk import adapters
+
+        tracker = adapters.HamiltonTracker(
+            project_id=41,  # modify this as needed
+            username="elijah@dagworks.io",
+            dag_name="ner-lancedb-pipeline",
+            tags={"context": "inference", "team": "MY_TEAM", "version": "1"},
+        )
+        adapter_list.append(tracker)
+
+    dr = build_driver(adapter_list)
+
+    r = dr.execute(["lancedb_result"], inputs={"query": query, "table_name": table_name})
+    print(r)
+
+
+def main():
```

Comment:
I believe `main()` should contain the logic and not rely on user inputs while `argparse` code exist after `if __name__ == "__main__"`.